gitcloner

```diff
@@ -32,5 +32,4 @@ def main():
     account.clone_repos()
 
 
-if __name__ == '__main__':
-    main()
+main()
```

Comment:
@fa7ad I modified the script to run individually. That's why I thought it was not that important. But I might be wrong of course. What do you think?